scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -598,7 +602,7 @@ def test_keepdims(hypotest, args, kwds, n_samples, n_outputs, paired, unpacker,
     if not unpacker:
         def unpacker(res):
             return res
-    rng = np.random.default_rng(0)
+    rng = np.random.RandomState(0)
```

Comment:
Why change from the more modern `default_rng` to `RandomState`?

Reply:
I changed it back, let's see if CI is happy. I assume this was done in the context of a test failure. A lot of tests rely on the exact random numbers used, and this looks like an alternative way of fixing such failures (changing the seed or bumping the tolerance are the other options).